alpaca/broker/client.py

```diff
@@ -414,3 +483,107 @@ def _parse_activity(data: dict) -> Union[TradeActivity, NonTradeActivity]:
             return parse_obj_as(TradeActivity, data)
         else:
             return parse_obj_as(NonTradeActivity, data)
+
+    # ############################## TRADE ACCOUNT DOCUMENTS ################################# #
+
+    def get_trade_documents_for_account(
+        self,
+        account_id: Union[UUID, str],
+        documents_filter: Optional[GetTradeDocumentsRequest] = None,
+    ) -> List[TradeDocument]:
+        """
+        Gets the list of TradeDocuments for an Account.
+
+        Args:
+            account_id (Union[UUID, str]): The id of the Account you wish to retrieve documents for. str values will
+              attempt to be upcast into UUID instances
+            documents_filter (GetTradeDocumentsRequest): The optional set of filters you can apply to filter the
+              returned list.
+
+        Returns:
+            List[TradeDocument]: The filtered list of TradeDocuments
+        """
+        account_id = validate_uuid_id_param(account_id)
+
+        result = self.get(
+            f"/accounts/{account_id}/documents",
+            documents_filter.to_request_fields() if documents_filter else {},
+        )
+
+        return parse_obj_as(List[TradeDocument], result)
+
+    def get_trade_document_for_account_by_id(
+        self,
+        account_id: Union[UUID, str],
+        document_id: Union[UUID, str],
+    ) -> TradeDocument:
+        """
+        Gets a single TradeDocument by its id
+
+        Args:
+            account_id (Union[UUID, str]): The id of the Account that owns the document
+            document_id (Union[UUID, str]): The id of the TradeDocument
+
+        Returns:
+            TradeDocument: The requested TradeDocument
```

Comment:
I believe there should be an empty line between the Returns and Raises sections.